source/includes/usage-examples/sample-write-application.java

```diff
@@ -0,0 +1,65 @@
+import com.mongodb.MongoException;
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.ServerApi;
+import com.mongodb.ServerApiVersion;
+import com.mongodb.bulk.BulkWriteResult;
+
+import com.mongodb.client.model.DeleteOneModel;
+import com.mongodb.client.model.InsertOneModel;
+import com.mongodb.client.model.ReplaceOneModel;
+import com.mongodb.client.model.UpdateOneModel;
+import com.mongodb.client.model.DeleteOptions;
+import com.mongodb.client.model.InsertManyOptions;
+import com.mongodb.client.model.InsertOneOptions;
+import com.mongodb.client.model.UpdateOptions;
+import com.mongodb.client.model.Updates;
+import com.mongodb.client.result.UpdateResult;
+import com.mongodb.client.result.DeleteResult;
+import com.mongodb.client.result.InsertManyResult;
+import com.mongodb.client.result.InsertOneResult;
+import com.mongodb.reactivestreams.client.MongoCollection;
+
+import org.bson.Document;
+
+import com.mongodb.reactivestreams.client.MongoClient;
+import com.mongodb.reactivestreams.client.MongoClients;
+import com.mongodb.reactivestreams.client.MongoDatabase;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.mongodb.client.model.Filters.eq;
+import static com.mongodb.client.model.Updates.set;
+
+public class updateDocument {
```

Comment:
_> why making this class not public is better in this case?_

One should not make program elements more accessible than needed. Doing so

- prevents program elements from being used by the code the author did not intent them to be used by;
- improves code readability, as a reader can easily see the boundary outside of which the program element is not used (in case of a package-access class, like here - the boundary if the package).<sup>(*)</sup>

In this case, the class that defines the `public static void main(String[])` method (and yes, the method must be `public`) does not need to be `public` itself.

<sup>(*)</sup> Unfortunately, Java SE still provides a mechanism allowing code to violate the boundaries set by access modifiers without author's consent, unless [JPMS](https://www.jcp.org/en/jsr/detail?id=376) is used. Nothing is perfect, and Java SE is definitely far from that.

Reply:
✅ Done.